tracking/run_tracking_inputs.py

```diff
@@ -0,0 +1,41 @@
+"""Run the tracked files and move into tracking/ directory."""
+
+import os
+from pathlib import Path
+import subprocess
+import shutil
+
+# Stellarator doesn't converge so don't want to track it
+exclusions = ["stellarator"]
```

Comment:
Can we run all the files anyway and modify the script to handle failure of an input file?